h/accounts/schemas.py

```diff
@@ -29,7 +29,7 @@ def get_blacklist():
         # can't load the file, then don't crash out, just log a warning about
         # the problem.
         try:
-            with open("h/accounts/blacklist", encoding="utf-8") as fp:
+            with codecs.open("h/accounts/blacklist", encoding="utf-8") as fp:
```

Comment:
Oh god there's no way you'd spot this when reading the code. Also, is this any different from just `open("h/accounts/blacklist")` now that `utf-8` is the default?